Final SEO updates (#1546)
pulumi/docs: content/blog/using-pulumi-with-aws-sqs-and-lambdas/index.md

authors: ["cyrus-najmabadi"]
tags: ["JavaScript","Serverless","AWS"]
date: "2018-07-10"

> meta_desc: "Learn how to use Amazon's SQS with Pulumi in order to post a Slack notification upon receipt of a message via SQS. This example Pulumi application demonstrates the ability to define your infrastructure directly in code (the AWS Queue, Lambda, and Event Subscriptions), instead of defining them separately."

Reviewer: ```suggestion
meta_desc: "Learn how to use Amazon's SQS with Pulumi in order to post a Slack notification upon receipt of a message via SQS. This Pulumi example Pulumi demonstrates the ability to define your infrastructure directly in code (the AWS Queue, Lambda, and Event Subscriptions), should you so desire."
```
